addons/event_booth_sale/wizard/event_booth_batch_configurator_line.py

```diff
@@ -0,0 +1,46 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import api, models, fields
+
+
+class EventConfiguratorLine(models.TransientModel):
+    """Event Booth Configuration"""
+    _name = "event.booth.editor.line"
+    _description = 'Edit Event Booth values on Sales Confirmation'
+    _order = "id desc"
+
+    editor_id = fields.Many2one('event.event.booth.configurator.batch')
+    product_id = fields.Many2one('product.product', string="Product", readonly=True)
+    event_booth_category_id = fields.Many2one(
+        'event.booth.category', string='Booth Category', required=True,
+        compute='_compute_event_booth_category_id', readonly=False, store=True)
+    event_booth_ids = fields.Many2many(
+        'event.booth', string='Booth', required=True,
+        compute='_compute_event_booth_ids', readonly=False, store=True)
+    event_id = fields.Many2one('event.event', string='Event', required=True)
+    event_booth_category_available_ids = fields.Many2many(related='event_id.event_booth_category_available_ids', readonly=True)
+    order_id = fields.Many2many('sale.order', string="Sale order")
+    sale_order_line_id = fields.Many2many('sale.order.line', string="Sale Order Line")
+
+    @api.depends('event_id')
+    def _compute_event_booth_category_id(self):
+        self.event_booth_category_id = False
+
+    @api.depends('event_id', 'event_booth_category_id')
+    def _compute_event_booth_ids(self):
```

Comment:
can we add a docstring to explain it act like an onchange ? (the objective is just to reset the value if we change the event / category)